README.md

```diff
@@ -341,6 +341,33 @@ See [ios/samples/README.md](./ios/samples/README.md) for more information.
 
 ### Windows
 
+#### Building on Windows with the Visual Studio 2019 compiler
+
+Install the following components:
+
+- [Visual Studio 2019](https://www.visualstudio.com/downloads)
+- [Python 3.7](https://www.python.org/ftp/python/3.7.0/python-3.7.0.exe)
+- [Cmake 3.13 or later](https://github.com/Kitware/CMake/releases/download/v3.13.4/cmake-3.13.4-win64-x64.msi)
+
+Open an appropriate Native Tools terminal for Visual Studio, for example the `Developer Command Prompt for VS2019`.
```

Comment:
Please mention that it needs to be a x64 terminal.

Reply:
thanks, done.